assets/app/views/browse/build-config.html

```diff
@@ -32,6 +31,9 @@
                     ><i class="fa fa-play fa-fw" aria-hidden="true"></i> Start Build</a>
                   </li>
                   <li>
+                    <a ng-href="{{buildConfig | editResourceURL}}" role="button"><i class="fa fa-pencil fa-fw" aria-hidden="true"></i>Edit</a>
```

Comment:
Add a space between the icon and the word Edit so the menu items line up correctly.